docs/catalog/cloud/containerize-your-workload-where-applicable.md

```diff
@@ -0,0 +1,36 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Containerize your workload where applicable
+
+## Description
+Containers may reduce unnecessary resource allocation and increase utilization of the compute resources, as they allow for bin packing and require less compute resources than virtual machines.
+
+## Solution
+Use [Draft](https://github.com/Azure/draft) to simplify application containerization by generating Dockerfiles and Kubernetes manifests.
```

Comment:
Vendor-specific implementations like Draft are not being accepted as patterns. Can this be presented in a vendor-agnostic way?

Reply:
Draft is actually OSS and works with any Kubernetes cluster (even though proposed by Microsoft). we could remove the link but i think it's a good enabler for customers

Reply:
and draft is meant as a proposal to help containrization (not the pattern per say)

Reply:
Even as OSS, it is still vendor-specific as it is using a specific tool/app/service/product as the solution. A solution for a pattern should be "use this tool" but rather "use this approach". The solution here should be focusing on how you would containerize workloads. Draft can be used as an example of a tool that makes it simpler, but the solution section of the pattern needs to talk about the concepts (creating dockerfiles, manifests, etc)